tests/python/gaia-ui-tests/gaiatest/visual_components/switch.py

```diff
@@ -0,0 +1,18 @@
+from marionette_driver import Wait
+from gaiatest.visual_components.visual_component import VisualComponent
+
+
+class Switch(VisualComponent):
```

Comment:
this would be great, btw, is there a reason to use js script rather than something like self.is_custom_element_checked(switch) (in gaia-switch) or self.is_element_checked(switch)?